### DISCUSSION TITLE
Compilation du fichier

### DISCUSSION THREAD

**Discussion by Wamavis** [texstudio-org/texstudio#1658] [General]
Bonjour tout le monde! Je suis un débutant en latex et j'utilise TexStudio comme éditeur. Je rencontre un problème au moment de la compilation de mon fichier tex. Voici le message d'erreur que cela m'affiche : "Sorry, but pdflatex.exe did not succeed. The log file hopefully contains the information to get MiKTeX going again: C:\Users\Lepetit\AppData\Local\MiKTeX\miktex\log\pdflatex.log". J'ai vraiment besoin de votre aide

**Comment by dbitouze** [comment]
**French :**
Bonjour. Cet espace de discussion est anglophone. Soit vous posez votre question en anglais, soit vous la posez sur un espace francophone, par exemple [ce site de questions et réponses](https://texnique.fr/osqa/). Quoiqu'il en soit, je ne suis pas certain que le problème soit lié à l'éditeur TeXstudio, mais plutôt à un défaut dans le fichier source `.tex`.

**English:**
Hello. This discussion space is English-speaking. Either you ask your question in English, or you ask it on a French-speaking space, for example [this site of questions and answers](https://texnique.fr/osqa/). Anyway, I am not sure that the problem is related to the TeXstudio editor, but rather to a defect in the `.tex` source file.

**Reply by Gomesz785** [reply]
@dbitouze the translation hehe  

Compiling File # 1658

Hello everyone! I am a latex newbie and use TexStudio as my editor. I have a problem while compiling my tex file. Here is the error message it shows me: "Sorry, but pdflatex.exe did not succeed. The log file hopefully contains the information to get MiKTeX going again: C: \ Users \ Lepetit \ AppData \ Local \ MiKTeX \ miktex \ log \ pdflatex.log ". I really need your help
